composer.json

```diff
@@ -77,5 +77,11 @@
     },
     "replace": {
         "zendframework/zend-expressive-authentication": "^1.1.0"
-    }
+    },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "git@github.com:geerteltink/laminas-coding-standard.git"
```

Comment:
@boesing you can add: 

```javascript
"no-api": true
```

I wrote a blog post about it https://samsonasik.wordpress.com/2018/10/07/using-no-api-key-on-github-forked-repository-in-composer-repositories-configuration-for-travis-build/
To fix the error.